Broken with latest wlroots

Probably related to: https://github.com/swaywm/wlroots/pull/1387

```
[57/58] Compiling C object 'waybox/waybox@@waybox@exe/main.c.o'.
FAILED: waybox/waybox@@waybox@exe/main.c.o
/nix/store/10yq7kwlvbc6h658izmrlsspry1g9f3c-gcc-wrapper-7.3.0/bin/cc -Iwaybox/waybox@@waybox@exe -Iwaybox -I../waybox -I../include -I/nix/store/w5r71zav1c4zmbi3p902bvl7b1zmyqyc-wayland-1.16.0/include -I/nix/store/yayjdd7fsghs2j1rm9rxsm99v4xd0k8m-wlroots-05bb44078667a11216611a8fbe87c5fe69e117f8/include -I/nix/store/lmv7cpljbqzbn0mxfpnk2knsm4c6bq12-pixman-0.34.0/include/pixman-1 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -O3 -Wno-unused-parameter -DWLR_USE_UNSTABLE -DWL_HIDE_DEPRECATED  -MD -MQ 'waybox/waybox@@waybox@exe/main.c.o' -MF 'waybox/waybox@@waybox@exe/main.c.o.d' -o 'waybox/waybox@@waybox@exe/main.c.o' -c ../waybox/main.c
In file included from ../waybox/main.c:7:0:
../include/waybox/server.h:21:10: fatal error: wlr/types/wlr_primary_selection.h: No such file or directory
 #include <wlr/types/wlr_primary_selection.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/include/waybox/server.h
+++ b/include/waybox/server.h
@@ -18,7 +18,7 @@
 #include <wlr/types/wlr_screenshooter.h>
 #include <wlr/types/wlr_matrix.h>
 #include <wlr/types/wlr_gamma_control.h>
-#include <wlr/types/wlr_primary_selection.h>
+#include <wlr/types/wlr_gtk_primary_selection.h>
 #include <wlr/types/wlr_output_layout.h>
 
 #include "waybox/output.h"

--- a/waybox/server.c
+++ b/waybox/server.c
@@ -60,7 +60,7 @@ bool start_wb(struct wb_server* server) {
 
     wlr_gamma_control_manager_create(server->wl_display);
 	wlr_screenshooter_create(server->wl_display);
-	wlr_primary_selection_device_manager_create(server->wl_display);
+	wlr_gtk_primary_selection_device_manager_create(server->wl_display);
 	wlr_idle_create(server->wl_display);
 
 	server->compositor = wlr_compositor_create(server->wl_display,

